templates/index2.html

```diff
@@ -0,0 +1,177 @@
+{% extends "layout.html" %}
```

Comment:
Is this file a repeat of index.html with just a minor difference around the header & link for the done section?

It would be much nicer to re-use index.html and have a jinja `{% if ... %}` block to control that.